phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -117,12 +118,15 @@
    * Class to represent pending data table rows
    */
   private static class PendingRow {
-      private boolean concurrent = false;
       private long count = 1;
+      private BatchMutateContext lastContext;
 
-      public void add() {
+      PendingRow(BatchMutateContext context) {
+          lastContext = context;
```

Comment:
slightly cleaner if we can initialize the count to 1 in the constructor itself.

Reply:
Will do that